docs/references/expo/offline-support.mdx

```diff
@@ -71,3 +71,45 @@ To enable offline support in your Expo app, follow these steps:
   }
   ```
 </Steps>
+
+## Handling network errors
+
+To handle network errors in your Expo app, you can use the `isClerkRuntimeError` function alongside the `code` property to identify the specific error. Here's an example of how you can handle network errors:
```

Comment:
```suggestion
To handle network errors in your Expo app, you can use the `isClerkRuntimeError` function and check the `code` property to identify the specific error. Here's an example of how you can handle network errors:
```

Reply:
Why is this showing a `network_error` error when its for offline support? This seems counter intuitive.

Reply:
Clerk custom flows should throw errors if there is no internet connection, e.g. `signIn.create()`. This error was muted until now and developers could not handle these.

Reply:
Please see the above about adding context to this section.